core/src/main/java/de/danoeh/antennapod/core/service/playback/LocalPSMP.java

```diff
@@ -317,12 +320,53 @@ private void resumeSync() {
                 float rightVolume = UserPreferences.getRightVolume();
                 setVolume(leftVolume, rightVolume);
 
+                int skipIntro = 0;
+                int skipEnd = 0;
+                if (media instanceof FeedMedia) {
+                    FeedMedia feedMedia = (FeedMedia) media;
+                    FeedPreferences preferences = feedMedia.getItem().getFeed().getPreferences();
+                    skipIntro = preferences.getFeedSkipIntro();
+                    skipEnd = preferences.getFeedSkipEnding();
+                }
+
                 if (playerStatus == PlayerStatus.PREPARED && media.getPosition() > 0) {
                     int newPosition = RewindAfterPauseUtils.calculatePositionWithRewind(
                         media.getPosition(),
                         media.getLastPlayedTime());
                     seekToSync(newPosition);
                 }
+                String skipMesg = "";
+                String skipIntroMesg = "";
+                if (playerStatus == PlayerStatus.PREPARED && media.getPosition() == 0) {
+                    if (skipIntro > 0) {
+                        int duration = getDuration();
+                        if (skipIntro * 1000 < duration) {
+                            seekToSync(skipIntro * 1000);
```

Comment:
Not sure if `LocalPSMP` is the right place to do this. What about Chromecast (`RemotePSMP`) or other media players that might get added in the future? I think this fits better into `PlaybackService` (but I am not sure where exactly).

Reply:
fixed in Playbackservice